contracts/AgglayerManager.sol

```diff
@@ -778,14 +774,7 @@ contract AgglayerManager is
         rollup.rollupVerifierType = rollupVerifierType;
 
         // Check verifier type
-        if (rollupVerifierType == VerifierType.Pessimistic) {
-            rollup.programVKey = programVKey;
-            rollup.lastPessimisticRoot = initPessimisticRoot;
-            rollup.lastLocalExitRoot = initRoot;
-            if (verifier.code.length == 0) {
-                revert InvalidVerifierAddress();
-            }
-        } else if (rollupVerifierType == VerifierType.ALGateway) {
```

Comment:
the condition  if (rollupVerifierType == VerifierType.Pessimistic) {
should revert with a custom error.
Now we can create a "weird" state transition rollup